frontend/components/panels/CollateralAuthorizationCheckPanel.vue

```diff
@@ -79,12 +71,6 @@ export default Vue.extend({
     },
     computed: {
         currentStateAndTitle(): PanelProps {
-            if (!this.walletAddress) {
-                return {
-                    name: 'inactive',
-                    title: `The ${this.collateralType} authorization is unknown until a wallet is connected`,
-                };
-            }
```

Comment:
Without connected wallet we can not get collateral auth status, as it's done per wallet. So this was actually correct

But there is no interdependency between `isCollateralAuthorized` and `isWalletAuthorized`, therefore my previous comment was to remove `isWalletAuthorized` from props and the logic, not `walletAddress`